app/scripts/modules/core/authentication/authentication.initializer.service.js

```diff
@@ -22,6 +35,10 @@ module.exports = angular.module('spinnaker.authentication.initializer.service',
         .error(loginRedirect);
     }
 
+    function loginNotification() {
+      notifierService.publish(`You have been logged out. <a role="button" class="action" onclick="document.location.reload(true)">Log in</button>`);
```

Comment:
The `onclick` event should just be:
```javascript
document.location.reload()
```
Passing `true` as a parameter will force the browser to reload everything - JS files, fonts, etc - from the server